hub/management/commands/import_hnh_polling_data.py

```diff
@@ -264,7 +372,10 @@ def extract_and_save_data(self):
                     data_types[col["slug"]] = data_type
 
             df = pd.read_csv(settings.BASE_DIR / "data" / file["source_filename"])
-
+            if file["data_set_name"] == "hnh_mrp_41-1":
```

Comment:
@alexander-griffen

1. If we’re skipping the first (empty) row for Question 41.1, shouldn’t we _also_ do it for Question 25 (which shares exactly the same formatting)?
2. The [pandas.read_csv() docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html) suggest`skip_blank_lines` defaults to `True`, so are you sure we need to skip the first (empty) row manually?

Reply:
I think you're right, this is unnecesary. I will remove it and rebase that in.